app/javascript/packs/application.js

```diff
@@ -10,7 +10,7 @@ import "channels";
 import * as PatientFormPage from "../Patient/New/PersonalDetails/PersonalDetails.bs";
```

Comment:
We should not import files that are not required application-wide in `applciation.js`

All the imports mentioned in this file will be loaded on all the pages.